src/unpack/fs.ts

```diff
@@ -6,11 +6,13 @@ import { map } from 'streaming-iterables'
 import { CarIndexedReader, CarReader } from '@ipld/car'
 import { CID } from 'multiformats'
 import { UnixFSEntry } from '@vascosantos/ipfs-unixfs-exporter'
+import { FsBlockStore } from '../blockstore/fs'
 
 // tslint:disable-next-line: no-var-requires needs types
 const toIterable = require('stream-to-it')
 
-import { unpack } from './index'
+import { unpack, unpackStream } from './index'
+import { Blockstore } from '../blockstore'
```

Comment:
```suggestion
import { Blockstore } from '../blockstore/index'
```

or ESM Node will break